server/services/placementService.test.ts

```diff
@@ -35,14 +46,34 @@ describe('PlacementService', () => {
   })
 
   describe('assignKeyworker', () => {
-    it('calls the assignKeyworker method of the placement client a response', async () => {
+    it('calls the assignKeyworker method of the placement client an returns a response', async () => {
       const assignKeyworker = cas1AssignKeyWorkerFactory.build()
       placementClient.assignKeyworker.mockResolvedValue({})
 
-      const result = await service.assignKeyworker(token, premisesId, placementId, assignKeyworker)
+      const result = await placementService.assignKeyworker(token, premisesId, placementId, assignKeyworker)
       expect(result).toEqual({})
       expect(placementClientFactory).toHaveBeenCalledWith(token)
       expect(placementClient.assignKeyworker).toHaveBeenCalledWith(premisesId, placementId, assignKeyworker)
     })
   })
+
+  describe('recordNonArrival', () => {
+    it('calls the recordNonArrival method of the placement client and returns a response', async () => {
+      const nonArrival = cas1NonArrivalFactory.build()
+      placementClient.recordNonArrival.mockResolvedValue({})
+      const result = await placementService.recordNonArrival(token, premisesId, placementId, nonArrival)
+      expect(result).toEqual({})
+      expect(placementClientFactory).toHaveBeenCalledWith(token)
+      expect(placementClient.recordNonArrival).toHaveBeenCalledWith(premisesId, placementId, nonArrival)
```

Comment:
Blank lines help to separate the Given/When/Then parts of tests to make them a bit more readable -- not a deal-breaker, but makes development a bit easier on the eye 🙂 

```suggestion
      const nonArrival = cas1NonArrivalFactory.build()
      placementClient.recordNonArrival.mockResolvedValue({})
      
      const result = await placementService.recordNonArrival(token, premisesId, placementId, nonArrival)
      
      expect(result).toEqual({})
      expect(placementClientFactory).toHaveBeenCalledWith(token)
      expect(placementClient.recordNonArrival).toHaveBeenCalledWith(premisesId, placementId, nonArrival)
```